.github/workflows/ci.yml

```diff
@@ -69,8 +69,14 @@ jobs:
         shell: bash
         run: |
           cd Consul.Test
-          ./consul agent -dev -config-file test_config.json &
+          ./consul agent -dev -config-file test_config.json -log-file consul.log >/dev/null &
```

Comment:
Do we need to redirect `stderr` as well ?

Reply:
Good catch, I will redirect `stdout` and `stderr` to different files — in case of failures, we'd rather more information than less.

Reply:
I think it would be even better to redirect them into the same file, e.g.: https://askubuntu.com/questions/625224/how-to-redirect-stderr-to-a-file

Reply:
I thought about it, but figured that it would make failure investigations easier to have them separate. What do you think?

Reply:
I don't have strong opinion here. Let's try as it is and see if it is helpful.